src/platform/platform_linux.cpp

```diff
@@ -29,10 +29,45 @@
 #include "logger/journald_logger.h"
 #include <disabled_update_prompt.h>
 
+#include <QStandardPaths>
+
+#include <cerrno>
+#include <cstring>
+#include <stdexcept>
+
 namespace mp = multipass;
 namespace mpl = multipass::logging;
 namespace mu = multipass::utils;
 
+namespace
+{
+constexpr auto autostart_filename = "multipass.gui.conditional-autostart.desktop";
```

Comment:
Just `.autostart.desktop`, we don't want to have to change that name when it's not conditional any more ;)

Reply:
Good point. My intention was to let users looking around know that this does not mean the gui is really autostarted. But that does not justify dealing with a lingering file later. I'll do that.